tests/integration/go/yaml/main.go

```diff
@@ -43,6 +44,14 @@ func main() {
 		hostIP := resources.GetResource("v1/Pod", "foo", "").(*corev1.Pod).Status.HostIP()
 		ctx.Export("hostIP", hostIP)
 
+		ct := resources.GetResource("stable.example.com/v1/CronTab", "my-new-cron-object", "")
+		cronSpec := ct.(*apiextensions.CustomResource).OtherFields.ApplyT(func(otherFields interface{}) string {
+			fields := otherFields.(map[string]interface{})
+			spec := fields["spec"].(map[string]interface{})
+			return spec["cronSpec"].(string)
+		})
+		ctx.Export("cronSpec", cronSpec)
+
```

Comment:
Am I right that there is not yet a test covering `NewCustomResource`?  That seems worth adding.